Sage/carbon
# chore(deps-dev): bump jest cli and some other deps to remove warnings

### Proposed behaviour


There are a very large number a minor and medium warnings, this pr fixes all the non-breaking ones, it reduces them from over 70k to just 2

### Current behaviour


There over 70k warnings raised

### Checklist


<del>- [ ] Unit tests<del>
<del>- [ ] Cypress automation tests<del>
<del>- [ ] Storybook added or updated<del>
<del>- [ ] Theme support<del>
<del>- [ ] Typescript `d.ts` file added or updated<del>

### Additional context


### Testing instructions

Main focus is to ensure the upgrade of `cypress` continues to work and then quick sanity check that the components still render in storybook and codesandbox
